commands/displayers/droplet_autoscale.go

```diff
@@ -0,0 +1,163 @@
+package displayers
+
+import (
+	"io"
+
+	"github.com/digitalocean/godo"
+)
+
+type DropletAutoscale struct {
+	AutoscalePools []*godo.DropletAutoscalePool         `json:"autoscale_pools"`
+	Droplets       []*godo.DropletAutoscaleResource     `json:"droplets"`
+	History        []*godo.DropletAutoscaleHistoryEvent `json:"history"`
+}
```

Comment:
These should be implemented as separate types rather than using the switch statements. This will allow you to remove the `format` flags that have been added manually. When the displayer is implemented correctly, the flag is added automatically. The help output will also have the column names populated automatically.